README.md

```diff
@@ -99,6 +99,27 @@ This Helm operator and chart are being developed to automate Falcon sensor deplo
 
 # Using Helm Operator
 
+## Deploy using the Operator Lifecycle Manager (OLM) deployment
+
+1. Install the Operator SDK CLI tool. See [https://sdk.operatorframework.io/docs/installation/](https://sdk.operatorframework.io/docs/installation/) for installation guide.
+
+2. Run the bundle
+   ```
+   operator-sdk run bundle quay.io/crowdstrike/falcon-helm-operator-bundle:latest
+   ```
+
+3. Apply the customized Helm operator resource:
+   ```
+   kubectl apply -f config/samples/crowdstrike_v1alpha1_falconsensor.yaml
+   ```
```

Comment:
Where was the step to customize this file?

Reply:
fixed